src/TruncateExtension.php

```diff
@@ -131,7 +131,15 @@ public function truncateLetters($html, $limit = 0, $ellipsis = "")
             }
         }
 
-        return $dom->saveHTML();
+        $renderHtml = "";
+        
+        $childNodes = ($body->childNodes->length > 1) ? $body->childNodes : $body->firstChild->childNodes;
+
+        foreach ($childNodes as $child) {
+            $renderHtml .= $dom->saveHTML($child);
+        }
+
+        return $renderHtml;
```

Comment:
We ought to refactor this out, and put this into a helper method so that truncateWords may benefit too.